### PR TITLE
fix(handling backbutton for queryparams): sdk was not re rendering wh…

### PR DESCRIPTION
**Pull request by manu-unbxd**
…en user click on back button

added popstate event to handle the backbutton when sdk is not in hashmode
ref: https://unbxddev.atlassian.net/browse/CS-5073

### CODE REVIEW

File: demo/src/pages/Search.js [hunk 1]
```diff
@@ -56,6 +56,7 @@ const Search = () => {
     };
 
     const handleRouteChange = (searchObj, hash, refreshId) => {
+        console.log(searchObj, hash, refreshId,"searchObj, hash, refreshId")
```
Comment (divyadaglia-unbxd) [inline demo/src/pages/Search.js:59]: Let's remove this console log

Comment (manu-unbxd) [reply demo/src/pages/Search.js:59]: its in the demo. so wont effect the sdk

Comment (divyadaglia-unbxd) [reply demo/src/pages/Search.js:59]: Since it is not useful, it is generally not a good practice to leave unnecessary code in the final branch.

Comment (manu-unbxd) [reply demo/src/pages/Search.js]: removed

**Pull request merged**